src/citrine/resources/data_objects.py

```diff
@@ -188,3 +188,10 @@ def validate_templates(self, *,
             if e.api_error is not None and e.api_error.validation_errors:
                 return e.api_error.validation_errors
             raise e
+
+    def filter_by_id(self, id_search_string) -> Iterator[DataObject]:
```

Comment:
Why are we calling this `filter_by`.  v2 eliminates that usage and relies on the `list_by` pattern.

Reply:
Sorry @kroenlein this should be in draft mode, but noted.